currency_exchange_rate_inverted/models/res_currency.py

```diff
@@ -0,0 +1,25 @@
+# -*- coding: utf-8 -*-
+# © 2015 Eficent
+# © 2015 Techrifiv Solutions Pte Ltd
+# © 2015 Statecraft Systems Pte Ltd
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+from openerp import api, fields, models
+
+
+class ResCurrency(models.Model):
+    _inherit = 'res.currency'
+
+    rate_inverted = fields.Boolean('Inverted exchange rate')
```

Comment:
Could this be a company dependent field?

- Company A has currency USD; exchange rates VND/USD doesn't need to be inverted
- Company B has currency VND (1USD=22000VND); exchange rate USD/VND needs to be inverted